src/js/components/Chart/__tests__/Chart-test.js

```diff
@@ -171,8 +188,54 @@ describe('Chart', () => {
     expect(tree).toMatchSnapshot();
   });
 
-  test('calcs', () => {
-    const result = calcs([1, 2, 3]);
+  test('calcs basic', () => {
+    const result = calcs([
+      [1, 2, 2],
+      [2, 2, 2],
+    ]);
+    expect(result).toMatchSnapshot();
+  });
+
+  test('calcs with single value', () => {
+    const result = calcs([1]);
+    expect(result).toMatchSnapshot();
+  });
+
+  test('calcs with negative min', () => {
+    const result = calcs([
+      [1, -2, -2],
+      [2, 2, 2],
+    ]);
+    expect(result).toMatchSnapshot();
+  });
+
+  test('calcs large thickness', () => {
```

Comment:
Why did you choose the term thickness? Chart has a `thickness` prop, so it might be a little confusing.